src/lib/PetitioResponse.ts

```diff
@@ -63,9 +63,11 @@ export class PetitioResponse {
 	}
 
 	/**
-	 * @return {*} The response body decoded as a UTF-8 string from the buffer.
+	 * @param {BufferEncoding} [encoding="utf8"] The encoding to use.
+	 * @return {string} The response body decoded as a UTF-8 (or whatever specified in `encoding`) string from the buffer.
+	 * @memberof PetitioResponse
```

Comment:
```suggestion
```
This is unnecessary, as TypeDoc can figure out which class this function belongs to.